mis_builder/migrations/8.0.2.0.0/post-migration.py

```diff
@@ -2,6 +2,27 @@
 # © 2017 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
+from openerp import api, SUPERUSER_ID
+
+
+def fill_defaults(env):
+    cr = env.cr
+
+    default_specs = {
+        'mis.report.instance.period': ['report_instance_id'],
+        'mis.report.kpi': ['report_id'],
+        'mis.report.query': ['report_id'],
+        'mis.report.instance': ['company_id'],
```

Comment:
Setting a default for company_id is going to be sensitive to the current default company of the admin user running the migration. In a multi-company context this is delicate. So I'm not sure we should set a default in the migration.